datahub/main.py

```diff
@@ -93,13 +94,15 @@ def create_opal_data(data: OpalDictData | OpalArrayData) -> dict[str, str]:
     return {"message": "Data submitted successfully."}
 
 
+# TODO: Fix return typing annotation
 @app.get("/opal")
-def get_opal_data() -> dict[str, float]:
+def get_opal_data() -> dict[str | Hashable, Any]:  # type: ignore[misc]
     """GET method function for getting Opal Dataframe as JSON.
 
     Returns:
         A Dict of the Opal Dataframe in JSON format
     """
-    # Requires opal-test branch to be merged
-    # return dt.opal_df.to_json()
-    return {"test": 0}
+    data = dt.opal_df.to_dict()
+    # can be converted back to DF with pd.DataFrame.from_json(data)
```

Comment:
This comment should be moved to the doctoring so it will be seen in the docs. Also should be changed to match the split orientation
```suggestion
    # can be converted back to DF with pd.DataFrame(**data)
```